x-pack/plugins/security_solution/public/hosts/components/hosts_table/index.tsx

```diff
@@ -176,8 +173,4 @@ const getNodeField = (field: HostsFields): string => {
     case HostsFields.lastSeen:
       return 'node.lastSeen';
   }
-  assertUnreachable(field);
```

Comment:
why are we removing that?

Reply:
Removed by accident, put it back now. Thank you!